src/elements/common/feature-checking/FeatureConsumer.tsx

```diff
@@ -0,0 +1,6 @@
+// @flow
```

Comment:
Is this and the other `// @flow` still needed?

Reply:
Drat. I copied these by hand, so this declaration snuck in. Will remove.